content/updates/index.json

```diff
@@ -1,4 +1,9 @@
 {
+  "2020-06": {
+    "title": "☀️ June Serenity updates",
+    "description": "This are the June Serenity updates",
```

Comment:
```suggestion
    "description": "In June, teams will be more empowered than ever! Check out our new product duplication feature, get a better user experience with rules engine updates and get more flexibility on asset naming conventions",
```

Reply:
and execute asset naming conventions more easily",

Reply:
replaced with "execute asset naming conventions with more flexibility" thanks!